collectors/HostSystemStatsCollector.py

```diff
@@ -1,53 +1,58 @@
 from BaseCollector import BaseCollector
-from prometheus_client.core import GaugeMetricFamily
 from tools.Resources import Resources
-from tools.helper import yaml_read
 from threading import Thread
 import os
 
 
 class HostSystemStatsCollector(BaseCollector):
+
     def __init__(self):
+        super().__init__()
+        self.vrops_entity_name = 'hostsystem'
         self.wait_for_inventory_data()
-        # self.post_registered_collector(self.__class__.__name__, self.g.name)
-
-    def describe(self):
-        yield GaugeMetricFamily('vrops_hostsystem_stats', 'testtext')
+        self.name = self.__class__.__name__
```

Comment:
I'd say we move this to __init__, then it is always there for sure

Reply:
This is a old version. All collectors are now using the `describe()` method from `Basecollector`